src/OpenTelemetry.AutoInstrumentation/FrameworkDescription.NetCore.cs

```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 // </copyright>
 
-#if NET6_0_OR_GREATER
```

Comment:
Not an issue, but, in principle this change is not needed anymore.

Reply:
Reverted.